worlds/sc2/options.py

```diff
@@ -971,11 +1006,27 @@ def get_disabled_campaigns(world: 'SC2World') -> Set[SC2Campaign]:
     return disabled_campaigns
 
 
+def get_disabled_flags(world: 'SC2World') -> MissionFlag:
+    excluded = 0
```

Comment:
You can initialize an empty intflag here with `excluded = MissionFlag(0)`. It also means you can just `return excluded` at the end rather than reconstructing a MissionFlag you could already have.